src/zope/interface/registry.py

```diff
@@ -184,6 +145,9 @@ def __init__(self, name='', bases=()):
         self._init_registrations()
         self.__bases__ = tuple(bases)
 
+        if hasattr(self, '_v_utility_registrations_cache'):
```

Comment:
OK. If I ever figure out what I was so worried about the first time around we can always come back to this.